taxonomies/ingredients.txt

```diff
@@ -79910,6 +79922,14 @@ fr:viande de poulet précuite
 nl:voorgekookt kippenvlees, gegaard kippenvlees
 # ingredient/fr:viande-de-poulet-précuite has 9 products in french @2019-04-16
 
+<en:chicken meat
+en:cooked chicken meat
+de:gegartes Hühnerfleisch
+
+<en:chicken meat
+en:cooked chicken meat with liquid seasoning
+de:gegartes Hühnerfleisch flüssig gewürzt
```

Comment:
Same here, instead of adding those 2 entries, we could just have a "with liquid seasoning" processing, (or simpler, we just make it a synonym of the en:seasoned processing:

in ingredients_processing.txt:

en:seasoned, with seasoning, with liquid seasoning